<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>113</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 45, 107);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>113</y>
     <width>1920</width>
     <height>967</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #e8e8e8; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>918</x>
     <y>14</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>145</y>
     <width>1856</width>
     <height>909</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="botonAgregar">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>175</y>
     <width>221</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 45, 107);
color: white;

border: 2px solid #002d6b;
border-radius: 15px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>36</y>
     <width>226</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>E-MOVILIDAD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>1700</x>
     <y>36</y>
     <width>126</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Usuario</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1836</x>
     <y>29</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>perfil-de-usuario.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>190</y>
     <width>707</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: icon-ui;
font-size:19px;</string>
   </property>
   <property name="text">
    <string>Baterías seguidas por telemetría: 6 (3 en línea)</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>250</y>
     <width>1892</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>958</x>
     <y>250</y>
     <width>3</width>
     <height>1054</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="botonBateria1">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>272</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 45, 107);
color: white;

border: 2px solid #002d6b;
border-radius: 15px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="botonBateria2">
   <property name="geometry">
    <rect>
     <x>374</x>
     <y>272</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 45, 107);
color: white;

border: 2px solid #002d6b;
border-radius: 15px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="botonBateria3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>272</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 45, 107);
color: white;

border: 2px solid #002d6b;
border-radius: 15px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="botonBateria4">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>532</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 45, 107);
color: white;

border: 2px solid #002d6b;
border-radius: 15px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="botonBateria5">
   <property name="geometry">
    <rect>
     <x>374</x>
     <y>532</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 45, 107);
color: white;

border: 2px solid #002d6b;
border-radius: 15px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="botonBateria6">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>532</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 45, 107);
color: white;

border: 2px solid #002d6b;
border-radius: 15px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>288</y>
     <width>67</width>
     <height>67</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;

border: 2px solid white;
border-radius: 25px; /* Bordes redondeados */
padding: 5px;
</string>
   </property>
   <property name="text">
    <string>#1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_36">
   <property name="geometry">
    <rect>
     <x>382</x>
     <y>288</y>
     <width>67</width>
     <height>67</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;

border: 2px solid white;
border-radius: 25px; /* Bordes redondeados */
padding: 5px;
</string>
   </property>
   <property name="text">
    <string>#1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_37">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>1020</y>
     <width>67</width>
     <height>67</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
color: black;

border: 2px solid white;
border-radius: 25px; /* Bordes redondeados */
padding: 5px;
</string>
   </property>
   <property name="text">
    <string>#1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>botonAgregar</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>botonBateria1</zorder>
  <zorder>botonBateria2</zorder>
  <zorder>botonBateria3</zorder>
  <zorder>botonBateria4</zorder>
  <zorder>botonBateria5</zorder>
  <zorder>botonBateria6</zorder>
  <zorder>label_35</zorder>
  <zorder>label_36</zorder>
  <zorder>label_37</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
